test/fixtures/app/index-with-ticker.html

```
@@ -0,0 +1,49 @@
+<html>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)